src/NuGet.Clients/NuGet.PackageManagement.UI/Options/CheckedListBoxAccessibleObject.cs

```diff
@@ -31,7 +32,16 @@ public override AccessibleObject GetChild(int index)
         {
             if (index >= 0 && index < CheckedListBox.Items.Count)
             {
-                var packageSource = (PackageSourceContextInfo)CheckedListBox.Items[index];
+                var item = (PackageSourceContextInfo)CheckedListBox.Items[index];
+                PackageSource packageSource = new PackageSource(item.Source, item.Name);
+                if (packageSource.IsHttp && !packageSource.IsHttps)
+                {
+                    var sourceMessage = string.Concat(
```

Comment:
I believe this helps us meet accessibility standards. Here's why: If I add a new source using the keyboard, even though I don't immediately hear the warning about HTTP sources, I do hear it as soon as it's added to the List when the item is announced.